app/src/main/kotlin/org/fossify/musicplayer/fragments/AlbumsFragment.kt

```diff
@@ -80,7 +81,10 @@ class AlbumsFragment(context: Context, attributeSet: AttributeSet) : MyViewPager
     }
 
     override fun onSearchQueryChanged(text: String) {
-        val filtered = albums.filter { it.title.contains(text, true) }.toMutableList() as ArrayList<Album>
+        val normalizedText = text.normalizeString()
+        val filtered = albums.filter { playlist ->
+            playlist.title.normalizeString().contains(normalizedText, true)
```

Comment:
Incorrect lambda parameter named `playlist` here (should be `album`). Same problem in every fragment.

Either use proper names or remove the custom parameter.


```suggestion
        val filtered = albums.filter {
            it.title.normalizeString().contains(normalizedText, true)
```